test/common/markdown/markdown-link.tests.ps1

```diff
@@ -98,6 +98,7 @@ Describe "Verify Markdown Links" {
                         # there could be multiple reasons why a failure is ok
                         # check against the allowed failures
                         $allowedFailures = [System.Net.HttpStatusCode[]](
+                            429, # Rate limited
```

Comment:
Do we need 429 in the allowed failures, now we are using the `-r` option?

Reply:
I'm going to experiment with this, it was just temporary to get things actually workng.

Reply:
Looks like it works